src/Stormpath/DataStore/DefaultDataStore.php

```diff
@@ -313,7 +313,15 @@ private function toStdClass(Resource $resource, $customData = false)
         {
             $customData = true;
         }
-        $propertyNames = $resource->getPropertyNames();
```

Comment:
Please bring this back in. Maybe calling it with the argument that I suggested below, according to which default value is better.